git_repo/services/ext/gitlab.py

```diff
@@ -21,6 +21,8 @@
 class GitlabService(RepositoryService):
     fqdn = 'gitlab.com'
 
+    _supports_nested_namespaces = True
```

Comment:
here I would change that to `max_nested_namespaces = 21`

as gitlab supports 21 subgroups maximum (if I remember right), so a good idea would be to set it up to that number!